src/browser.js

```diff
@@ -148,35 +204,20 @@ function save(namespaces) {
  */
 
 function load() {
-  try {
-    return exports.storage.debug;
-  } catch(e) {}
-
-  // If debug isn't set in LS, and we're in Electron, try to load $DEBUG
-  if (typeof process !== 'undefined' && 'env' in process) {
-    return process.env.DEBUG;
+  if (exports.storage) {
+    exports.storage.get('debug', exports.enable);
```

Comment:
In other words, you're pointing out point no. 2 in the description of this PR

> in browsers apps/extensions, due to the async nature, calling debug before we read from browser.storage.local may end up in lost messages

Comments:

1. our workaround is to queue up messages, and process them after the local storage value is read
2. I see this fixed as part of another issue/PR (the goal is to support chrome.storage.local properly now)
3. the risk is rather small that messages are lost, given that the main execution is deferred (setTimeout 0)